src/App.tsx

```diff
@@ -330,9 +330,14 @@ any,
       }
     }
       await this.repositoryBrowser?.current?.initAsync(this.lfRepoTreeService!, focusedNode);
+      if (this.repositoryBrowser?.current) {
+        this.repositoryBrowser.current.isSelectable = this.isNodeSelectable;
+        this.repositoryBrowser.current.addEventListener('entrySelected', this.onEntrySelected );
+      }
     }
 
-  isNodeSelectable = (node: LfRepoTreeNode) => {
+  isNodeSelectable = async (treenode: LfTreeNode) => {
+    const node = treenode as LfRepoTreeNode;
```

Comment:
RecordSeries should be selectable too?

Reply:
I thought recordseries only has recordfolders or recordseries. can we save a document to a recordseries?

Reply:
try it?